executor/README.md

```
@@ -152,6 +165,12 @@ Execute code in a new sandbox.
 }
 ```
 
+**Request requirements:**
+- Include `X-API-Key` on executor requests.
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)